andela/titan-dev-ah-frontend
# 163518666 Liking a comment

#### What does this PR do?
- Like a  comment
#### Description of Task to be completed?
- Enable liking of a comment of an article
#### How should this be manually tested?
- clone the repos and check out the branch `ft-liking-a-comment-163518666`
- clone the repos and check out the branch `develop`
- Run `yarn` to install all dependencies on both branches
- Run `yarn test` to run tests, they should all pass on both branch
- Start your backend server on the address `localhost:3000/` by issuing `yarn start` command
- Create some articles using client tool like the postman 
- Grab slug of one of the articles you created. 
- Run `yarn start` to start your server
- Navigate to `localhost:8080/articles/:{slug}`
- Like on a comment you should see the color of the heart icon change under the article.
#### Any background context you want to provide?
- None
#### What are the relevant pivotal tracker stories?
# 163518649
